hydroflow_plus_test/src/cluster/paxos_bench.rs

```diff
@@ -54,7 +59,8 @@ pub fn paxos_bench<'a>(
             new_leader_elected_complete.complete(
                 new_leader_elected
                     .broadcast_bincode(&clients)
-                    .map(q!(|(leader_id, _)| leader_id)),
+                    .map(q!(|(leader_id, _)| leader_id))
+                    .assume_ordering(), // TODO(shadaj): this is a bug, if we switch leaders rapidly, message reordering could cause a client to have the wrong one
```

Comment:
We can just modify the leader election message to include the leader's ballot, and the client just keeps the one with the highest ballot